doc/user/spec_examples.md

```diff
@@ -121,3 +121,26 @@ spec:
     backupClusterName: "cluster-a"
     storageType: "PersistentVolume"
 ```
+
+### Three members TLS-secured cluster
+
+(TODO: client TLS)
+
+First, create secret "etcdtls":
+```
+$ kubectl create secret generic etcdtls --from-file=ca-peer-crt.pem --from-file=peer-crt.pem --from-file=peer-key.pem
+```
+
+Set "etcdtls" as server secret:
+
+```yaml
+apiVersion: "etcd.coreos.com/v1beta1"
+kind: "Cluster"
+metadata:
+  name: "example-etcd-cluster"
+spec:
+  size: 3
+  TLS:
+    static:
+      serverSecretName: etcdtls
```

Comment:
Although this implements peer TLS only, this secret should also contain server's client TLS, right? We will implement that later